rbac/management/permission/model.py

```diff
@@ -26,6 +26,7 @@ class Permission(models.Model):
     resource_type = models.TextField(null=False)
     verb = models.TextField(null=False)
     permission = models.TextField(null=False, unique=True)
+    description = models.TextField(default="")
```

Comment:
This is failing for me with a not null constraint when I run seeds. Seems like you were right with needing `null=True`? 😄